ocaml/runtime/intern.c

```diff
@@ -427,6 +427,15 @@ static void intern_rec(value *dest)
         intern_cleanup();
         caml_failwith("input_value: integer too large");
         break;
+#endif
+      case CODE_UNBOXED_INT64:
```

Comment:
The code looks fine, but is this actually tested anywhere?

Reply:
Yes, the testsuite doesn't pass if this marshalling is incorrect.